### PR TITLE
Add PT-PT Language for frontend

### PR DESCRIPTION
**Pull request by pedrommvv**
Adding new translation for Photoprism 💯 

Awesome project!

### CODE REVIEW

**Comment by graciousgrey** [comment]
Thank you :) Would you like to add backend translations as well (https://docs.photoprism.org/developer-guide/translations/#add-new-translation_1) ?

**Comment by graciousgrey** [comment]
We copied the pt_BR backend translations to assets/locales/pt/default.po until we have the pt-pt translations for the backend. The frontend translation will be part of the next release :)

File: frontend/src/locales/pt_PT.po [hunk 1]
```diff
@@ -0,0 +1,2367 @@
...
+msgid "Added"
+msgstr "Adicionado"
+
+#: src/pages/settings.vue:52
+msgid "Advanced"
+msgstr "Avançado"
+
+#: src/options/options.js:281
+msgid "After 1 day"
+msgstr "Depois de 1 dia"
```
Comment (ptalmeida) [inline frontend/src/locales/pt_PT.po:115]: Wouldn't "Após" be more correct?

File: frontend/src/locales/pt_PT.po [hunk 2]
```diff
@@ -0,0 +1,2367 @@
...
+msgid "Daily"
+msgstr "Diariamente"
+
+#: src/dialog/photo/details.vue:118
+msgid "Day"
+msgstr "Dia"
+
+#: src/pages/settings/advanced.vue:76
+msgid "Debug Logs"
+msgstr "Registros de depuração"
```
Comment (ptalmeida) [inline frontend/src/locales/pt_PT.po:557]: Registros -> Registos

File: frontend/src/locales/pt_PT.po [hunk 3]
```diff
@@ -0,0 +1,2367 @@
...
+msgid "Feedback"
+msgstr "Dê a sua opinião"
+
+#: src/model/file.js:242
+msgid "File"
+msgstr "Ficheiro"
+
+#: src/routes.js:240
+msgid "File Browser"
+msgstr "Explorador de Arquivos"
```
Comment (ptalmeida) [inline frontend/src/locales/pt_PT.po:861]: Why "Arquivos" and not "Ficheiros"?

File: frontend/src/locales/pt_PT.po [hunk 4]
```diff
@@ -0,0 +1,2367 @@
...
+"para evitar duplicações."
+
+#: src/pages/settings/general.vue:229
+msgid "Imported files will be sorted by date and given a unique name."
+msgstr ""
+"Ficheiros importados serão organizados por data e receberão um nome único."
+
+#: src/pages/library/import.vue:5
+msgid "Importing %{name}…"
+msgstr "Importando %{name}…"
```
Comment (ptalmeida) [inline frontend/src/locales/pt_PT.po:1020]: A Importar

File: frontend/src/locales/pt_PT.po [hunk 5]
```diff
@@ -0,0 +1,2367 @@
...
+msgid "Token"
+msgstr "Token"
+
+#: src/options/options.js:241
+msgid "Topographic"
+msgstr "Topográfico"
+
+#: src/pages/about/about.vue:47
+msgid "Trademarks"
+msgstr "Marcas registradas"
```
Comment (ptalmeida) [inline frontend/src/locales/pt_PT.po:2123]: registradas -> registadas

File: frontend/src/locales/pt_PT.po [hunk 6]
```diff
@@ -0,0 +1,2367 @@
...
+"Por favor, entre em contato via hello@photoprism.app se tiver dúvidas ou "
+"precisar de ajuda."
+
+#: src/dialog/upload.vue:45
+msgid "Please don't upload photos containing offensive content."
+msgstr "Por favor, não envie fotos com conteúdo ofensivo."
+
+#: src/options/options.js:126
+msgid "Polish"
+msgstr "Polonês"
```
Comment (ptalmeida) [inline frontend/src/locales/pt_PT.po:1607]: Polonês -> Polaco

**Review by ptalmeida — COMMENTED** [review]
Good job, added some comments of my own. I believe Gerund should not be used however, with the infinitive taking it's place.

imagem

**Pull request merged**